src/components/icons/StrikethroughIcon.tsx

```diff
@@ -1,42 +1,38 @@
-import { css, cx } from '../../../styled-system/css'
-import { icon } from '../../../styled-system/recipes'
-import { token } from '../../../styled-system/tokens'
-import IconType from '../../@types/Icon'
-import { ICON_SCALING_FACTOR } from '../../constants'
-
-/** Strikethrough icon. */
-const StrikethroughIcon = ({ fill, size = 20, style = {}, cssRaw }: IconType) => {
-  const newSize = size * ICON_SCALING_FACTOR
-  const strokeColor = style.fill || fill || token('colors.fg')
+import IconType from '../../@types/IconType'
+import AnimatedIcon from './AnimatedIcon'
+import animationData from './animations/16-strikethrough_3.json'
 
+/** Strikethrough Icon with Conditional Lottie Animation. */
```

Comment:
The original bounce animation was implemented to cancel early if the user drags the cursor away from the button (cancelling the action). @raineorshine, should the Lottie animations also cancel early in this case? Currently, the bounce animation cancels but the Lottie animation does not:

https://github.com/user-attachments/assets/121a74e9-1202-47a7-8a58-f4eef1044114

Reply:
The lottie animations no longer activate in this case, as expected 👍